src/Microsoft.ComponentDetection.Detectors/uv/UvLockComponentDetector.cs

```diff
@@ -0,0 +1,76 @@
+namespace Microsoft.ComponentDetection.Detectors.Uv
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.ComponentDetection.Contracts;
+    using Microsoft.ComponentDetection.Contracts.Internal;
+    using Microsoft.ComponentDetection.Contracts.TypedComponent;
+    using Microsoft.Extensions.Logging;
+
+    public class UvLockComponentDetector : FileComponentDetector
```

Comment:
We'll want to make sure to initially have this with the DefaultOffComponentDetector interface to test prior to enabling in prod

Reply:
https://github.com/microsoft/component-detection/blob/main/docs/creating-a-new-detector.md#detector-lifecycle